.github/workflows/reuse.yml

```diff
@@ -0,0 +1,254 @@
+name: If Levaithan was reusuable?
+
+on:
+  workflow_call:
+    inputs:
+      LEVIATHAN_REF:  # id of input
+        description: "Leviathan branch, or tag, or commit to test with. Examples: 'refs/heads/<branchName>', 'refs/tags/<tagName>', '<commitId>'"
+        required: false
+        type: string
+        default: 'refs/heads/master'
+      # CONFIG:
+      #   description: 'path to config.js'
+      #   required: false
+      #   default: './suites/config.js'
+      SUITES:
+        description: 'path to test folders'
+        required: false
+        type: string
+        default: './suites'
+      # TEST_SUITE_NAME:
+      #   description: 'which test suite to run'
+      #   required: false
+      #   type: string
+      WORKSPACE:
+        description: 'path to WORKSPACE'
```

Comment:
What is WORKSPACE in this context?

Reply:
Jenkins equivalent of `.levaithan-inputs` folder. 
A new folder that houses suites, config, reports for them to be bind mounted correctly to the Client container.

Reply:
I don't think it needs to be an input then